templates/helpers.rb

```diff
@@ -314,6 +316,47 @@ def in_context name
     @convert_context.pop
     result
   end
+
+  STEM_EQNUMS_AMS = 'ams'
+  STEM_EQNUMS_NONE = 'none'
+  STEM_EQNUMS_VALID_VALUES = [
+    STEM_EQNUMS_NONE,
+    STEM_EQNUMS_AMS,
+    'all'
+  ]
+
+  # Generate the Mathjax markup to process STEM expressions
+  # @param cdn_base [String]
+  # @return [String]
+  def generate_stem(cdn_base)
+    if attr?(:stem)
+      eqnums_val = attr('eqnums', STEM_EQNUMS_NONE)
+      eqnums_val = STEM_EQNUMS_AMS if eqnums_val == ''
+      unless STEM_EQNUMS_VALID_VALUES.include?(eqnums_val)
+        eqnums_val = STEM_EQNUMS_AMS
+      end
+      mathjax_configuration = {
+        tex: {
+          inlineMath: [Asciidoctor::INLINE_MATH_DELIMITERS[:latexmath]],
+          displayMath: [Asciidoctor::BLOCK_MATH_DELIMITERS[:latexmath]],
+          processEscapes: false,
+          tags: eqnums_val,
+        },
+        options: {
+          ignoreHtmlClass: 'nostem|nolatexmath'
+        },
+        asciimath: {
+          delimiters: [Asciidoctor::BLOCK_MATH_DELIMITERS[:asciimath]],
+        },
+        loader: {
+          load: ['input/asciimath', 'output/chtml', 'ui/menu']
+        }
+      }
+      mathjaxdir = attr('mathjaxdir', "#{cdn_base}/mathjax/3.2.0/es5")
```

Comment:
I think we should extract MathJax version as a constant (to make it easier to bump the version).

Reply:
Done